api/Window.json

```diff
@@ -1041,11 +1041,11 @@
             },
             "safari": [
               {
-                "version_added": "6.1"
+                "version_added": "7"
               },
               {
                 "version_added": "6",
-                "version_removed": "6.1",
+                "version_removed": "7",
```

Comment:
This case is a little bit irregular since there's no corresponding prefixed entry for safari_ios. There probably should be.

Reply:
http://mdn-bcd-collector.appspot.com/tests/api/Window/webkitCancelAnimationFrame reveals there's more wrong here, the API is still supported!

I'll back out the change from this PR.